scripts/python/src/fodt/keyword_linker.py

```diff
@@ -0,0 +1,161 @@
+import logging
+import re
+import xml.sax
+import xml.sax.handler
+import xml.sax.xmlreader
+import xml.sax.saxutils
+
+from pathlib import Path
+
+import click
+
+from fodt.constants import ClickOptions, Directories, FileNames, FileExtensions
+from fodt.exceptions import HandlerDoneException, ParsingException
+from fodt import helpers
+from fodt.xml_helpers import XMLHelper
+
+class ExtractURI_Handler(xml.sax.handler.ContentHandler):
+    def __init__(self, keyword_name: str) -> None:
+        self.keyword_name = keyword_name
+        self.in_section = False
+        self.in_bookmark = False
+        self.in_bookmark2 = False
+        self.uri = ""
+
+    def characters(self, content: str):
+        if self.in_bookmark2:
+            # Check if the content matches the keyword name
+            match =  re.search(self.keyword_name, content)
```

Comment:
I don't think that this enough. Aren't there keywords that include other keywords in their name? Hence we need an exact match. Maybe search for "^"+self.keyword_name+"$"`, or something?

Reply:
Looking at the other PR, I seem to be wrong, though. Would you please enlighten me why/how?

Reply:
> I don't think that this enough.

@blattms Actually the regex should in theory not be necessary, but I kept it there as a control if something weird  happened. Since we already know that we are inside the file with the given keyword name and we are in the first  heading with outline level 3, it should be enough to extract the URI link from the first `<text:bookmark-start>` tag. Anyway, matching the keyword name exactly was also abondoned due to the other issue with span tags that might occur in the middle of the keyword and since it in theory should be necessary to check for it as mentioned above.